app/build.gradle

```diff
@@ -110,7 +110,6 @@ android {
         minSdkVersion 19
         targetSdkVersion 28
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        testInstrumentationRunnerArguments clearPackageData: 'true'
```

Comment:
Will all tests pass if we don't clear data before every test?

Reply:
Yes, they all work fine now as we've removed the temperamental ones, and the emulator starts it from a clean state at the beginning.